test/e2e/tests/metrics/delete-metametrics-data.spec.ts

```diff
@@ -202,6 +206,10 @@ describe('Delete MetaMetrics Data @no-mmi', function (this: Suite) {
         const deleteMetaMetricsDataButtonRefreshed = await driver.findElement(
           rowLocators.deleteMetaMetricsDataButton,
         );
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        await (deleteMetaMetricsDataButtonRefreshed as any).waitForElementState(
+          'disabled',
+        );
         assert.equal(
           await deleteMetaMetricsDataButtonRefreshed.isEnabled(),
```

Comment:
if we just did this directly, it introduces a race condition where we get the element state as enabled, before we give it time to be disabled, so we need to wait for the condition we want. Then this assert is redundant, but also doesn't do any harm